documentation/asciidoc/computers/configuration/raspi-config.adoc

```diff
@@ -232,11 +232,11 @@ Configure the network's proxy settings.
 
 ===== Boot Order
 
-On the Raspberry pi 4, you can specify whether to boot from USB or network if the SD card isn't inserted. See xref:raspberry-pi.adoc#raspberry-pi-4-bootloader-configuration[this page] for more information.
+On the Raspberry Pi 4, you can specify whether to boot from USB or network if the SD card isn't inserted. See xref:raspberry-pi.adoc#raspberry-pi-4-bootloader-configuration[this page] for more information.
 
 ===== Bootloader Version
 
-On the Raspberry pi 4, you can tell the system to use the very latest boot ROM software, or default to the factory default if the latest version causes problems.
+On the Raspberry Pi 4, you can tell the system to use the very latest boot ROM software, or default to the factory default if the latest version causes problems.
```

Comment:
@lurch 
This means probably `factory reset`.

Reply:
No: it means use the EEPROM image from the default branch. There is no factory reset option, since every time you update the EEPROM image, what you are actually doing is completely rewriting the contents of the EEPROM.